session_0/word_processing.rb

```diff
@@ -18,7 +24,13 @@ def lower_case!(words)
 # words_with_prefix('apple', ['apple', 'ball', 'applesauce']) would
 # return the words 'apple' and 'applesauce'.
 def words_with_prefix(prefix, words)
-  raise NotImplementedError # TODO
+  nam = Array.new
+  words.each do |word|
```

Comment:
Can the select function be used here? https://www.rubyguides.com/2019/04/ruby-select-method/